Base/usr/share/man/man1/Applications/Maps.md

```diff
@@ -0,0 +1,34 @@
+## Name
+
+![Icon](/res/icons/16x16/app-maps.png) Maps
+
+[Open](file:///bin/Maps)
+
+## Synopsis
+
+```**sh
+$ Maps
+```
+
+## Description
+
+`Maps` is an application for browsing a map of the world.
+
+The default data provider is [OpenStreetMaps](https://www.openstreetmap.org/), an open-source community-maintained map similar in spirit to Wikipedia.
+
+## Features
+Pan around the map by clicking and dragging.
+
+Zoom-in and out by using the toolbar icons, the mouse wheel or `Ctrl +` to zoom-in and `Ctrl -` to zoom-out. Reset the zoom to a global view with `Ctrl 0`.
+Double-clicking anywhere on the map zooms-in to that location. Double-clicking whilst holding `Shift` zooms-out.
+
+Right-click on a location to:
+* Copy its coordinates
+* Open it in OpenStreetMap, Google Maps or Bing Maps
```

Comment:
I would change this to: "Open it in various other map services / websites", because otherwise we need to update the man page every time we add or remove some other "Open in ..." button.

Reply:
Sure